packages/elements/src/list/index.ts

```diff
@@ -353,22 +342,21 @@ export class List<T extends DataItem = ItemData> extends ControlElement {
    */
   protected highlightItem (item?: T, scrollToItem = false): void {
     if (item) {
-      const elementToFocus = this.elementFromItem(item);
-      const focus = this.activeElement && this.activeElement !== elementToFocus;
       this.clearHighlighted();
       this.composer.setItemPropertyValue(item, 'highlighted', true);
-      focus && elementToFocus?.focus({ preventScroll: true });
+      (this.tabIndex >= 0 && this.renderer.key && item.value) && this.setAttribute('aria-activedescendant', `${this.renderer.key}-${item.value}`);
```

Comment:
Would be nice to make this more readable by moving the key generation to a helper.

```suggestion
      const activeKey = getItemKey(this.renderer.key, item.value);
      this.tabIndex >= 0 && activeKey && this.setAttribute('aria-activedescendant', activeKey);
```